addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -416,7 +425,10 @@ const ColorPaletteWidget = Widget.extend({
      */
     _onSwitchPaneButtonClick(ev) {
         ev.stopPropagation();
-        this.el.querySelectorAll('.o_we_colorpicker_switch_pane_btn').forEach(el => {
+        this.el.querySelectorAll('.o_we_colorpicker_switch_pane_btn').forEach((el, index) => {
+            if (el === ev.currentTarget) {
+                this.selectedTab = index;
```

Comment:
Does not seem useful to keep track of which tab is selected right? Remove the code if not useful and the `selectTab` internal variable too in that case.